deployments/ansible/roles/collector/tasks/otel_win_install.yml

```diff
@@ -0,0 +1,53 @@
+---
+# Fetching the latest version of splunk-otel-collector
+- name: Get splunk-otel-collector latest version
+  ansible.windows.win_get_url:
+    url: "{{win_base_url}}/splunk-otel-collector/msi/{{package_stage}}/latest.txt"
+    dest: "%TEMP%"
+  register: latest
+  when: splunk_otel_collector_version == "latest"
+
+- name: get content of remote file
+  slurp:
+    src: "{{latest.dest}}"
+  register: version
+
+- name: decode remote file content for latest version
+  set_fact:
+    splunk_otel_collector_version: "{{version.content | b64decode }}"
+  when: splunk_otel_collector_version == "latest"
+
+# Download msi package manager for Windows
+- name: Get splunk-otel-collctor for windows
+  ansible.windows.win_get_url:
+    url: "{{win_base_url}}/splunk-otel-collector/msi/{{package_stage}}/{{msi_name}}"
+    dest: "%TEMP%"
+  register: otel_msi_package
+
+# Check for existing splunk-otel-collector Installation
+- name: Check if splunk-otel-collector service is installed
+  ansible.windows.win_service:
+    name: splunk-otel-collector
+  register: service_info
+
+# Uninstall existing splunk-otel-collector service
+- name: Delete previous splunk-otel-collector service if exist
+  ansible.windows.win_service:
+    name: splunk-otel-collector
+    state: absent
+  when: service_info.exists
+
+# Install latest version of Splunk-otel-collector on Windows
+- name: Install splunk-otel-collector-msi
+  ansible.windows.win_package:
+    path: "{{otel_msi_package.dest}}"
+    state: present
+
+# Push Custom Configuration file for splunk-otel-collector on Windows.
+- name: Push Custom Config file for splunk-otel-collector, If provided
+  ansible.windows.win_copy:
+    src: "{{splunk_otel_collector_config_source}}"
+    dest: "{{splunk_otel_collector_config}}"
+    remote_src: yes
```

Comment:
It shouldn't be `remote_src`. `splunk_otel_collector_config_source` is meant to upload a local custom config to a remote host